tools/add_triage_signature.py

```diff
@@ -839,6 +839,56 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket(report)
 
 
+class DownloadImage(ErrorSignature):
+    image_download_regex = re.compile(r"Host (.+?): New image status (.+?). result:.+?; download rate: (.+?) MBps")
```

Comment:
let's use named groups, it makes it much more readable to parse it afterwards:
```suggestion
    image_download_regex = re.compile(r"Host (?P<hostname>.+?): New image status (?P<image>.+?). result:.+?; download rate: (?P<download_rate>.+?) MBps")
```